### ISSUE TITLE
Prisma2: BatchPayload type not generated 

### ISSUE THREAD

**Issue by iherger** [graphql-nexus/nexus-plugin-prisma#341]
It seems that the `BatchPayload` type is not automatically generated for Prisma2/nexus projects.

In Prisma1, it was automatically generated as 
```typescript
type BatchPayload {
  count: Long!
}
```

**Comment by iherger** [comment]
Not sure if this is `type/feature` issue or a `type/bug` issue.

Nexus/nexus-prisma automatically generates the type `BatchPayload` in the GraphQL schema, but not in the typegen output, i.e. 
```ts
  outputs: {
    typegen: join(__dirname, '../generated/nexus-typegen.ts'), // BatchPayload missing
    schema: join(__dirname, '/schema.graphql'),                // BatchPayload present
  },
```
So, you basically have to define `BatchPayload` manually as 
```ts
export const BatchPayload = objectType({
  name: 'BatchPayload',
  definition(t) {
    t.int('count');
  },
});
```
and everything works alright.

Feel free to close this issue if that's the desired behaviour.

**Comment by jasonkuhrt** [comment]
Fixed by #441
